doc/environment_variables.md

```diff
@@ -40,6 +40,14 @@ some configuration as environment variables that can be set.
     fallback engine when nothing better exists
   - legacy - the (deprecated) original polling engine for gRPC
 
+* GRPC_EVENTENGINE_STRATEGY
```

Comment:
Done. Given the discussions last week around having multiple implementations in the OSS gRPC repo, I think this might be an ok mechanism to let users choose which is used (libuv vs possibly another on linux, for example). But there are other ways already, and we can defer a decision at least.